core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -627,8 +699,36 @@ public void setProperties(Map<String, String> properties) {
    * Changes posix file permission.
    *
    * @param path path of the file
-   * @param posixPerm standard posix permission like "777", "775", etc
+   * @param permission the permission to set in short format, e.g. 0777
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract void setPermission(String path, short permission) throws IOException;
+
+  /**
+   * Gets the owner of the given path. An empty implementation should be provided if not supported.
+   *
+   * @param path path of the file
+   * @return the owner of the file
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract String getOwner(String path) throws IOException;
```

Comment:
Thanks for pointing this out. Once your [rename PR] (https://github.com/Alluxio/alluxio/pull/3562/files) is merged, I'll updated this one.